akka-stream/src/main/scala/com/typesafe/sslconfig/akka/SSLEngineConfigurator.scala

```diff
@@ -21,6 +21,7 @@ final class DefaultSSLEngineConfigurator(
     enabledProtocols: Array[String],
     enabledCipherSuites: Array[String])
     extends SSLEngineConfigurator {
+  config ne null // @unused unavailable
```

Comment:
adding `@unused` on the variable didn't work here? OK though.

Reply:
Yea, not available because this is not in `_root_.akka`